### DISCUSSION TITLE
New system install and new zim install not seeing notebook folder

### DISCUSSION THREAD

**Discussion by darylbubba** [zim-desktop-wiki/zim-desktop-wiki#2023] [General]
I glitched my system and then found that my backups were no good. Luckily I suspected my incompetence and copied my home folder to a seperate drive. Now I have reinstalled Ubuntu and copied my Zim "notebook" folder back into my "home" directory. I then reinstalled Zim from synaptic and ran it. Zim does not list my notebooks and simply wants me to start a new notebook. How can I make Zim see my notebooks and make them usable?

**Comment by introt** [comment]
Hi! Zim doesn't search for notebooks so you'll have to tell it where to look. This can be done via the GUI from the Add Notebook dialog, just type a path to an existing notebook:

![image](https://user-images.githubusercontent.com/26069787/165244912-d138f19b-a7ed-48a9-884f-b8c3fe3c239a.png)

Alternatively, if you can't restore it from your backup, you can edit `~/.config/zim/notebook.list` by hand; the format is as follows:

```
[NotebookList]
Default=
1=~/Notebooks/Notes
2=~/Notebooks/Testing

[Notebook 1]
uri=~/Notebooks/Notes
name=Notes
interwiki=notes
icon=None

[Notebook 2]
uri=~/Notebooks/Testing
name=Testing
interwiki=testing
icon=None
```

Hope this helps! Remember to test your backups regularly in the future :)
